packages/templates/cra-template-web-viewer/template.json

```diff
@@ -1,58 +1,73 @@
 {
   "package": {
     "dependencies": {
-      "@bentley/icons-generic": "^1.0.13",
-      "@bentley/react-scripts": "^5.0.4",
-      "@itwin/core-telemetry": "^3.7.0",
-      "@itwin/appui-abstract": "^3.7.0",
+      "@bentley/icons-generic": "1.0.13",
```

Comment:
why are deps being pinned here?, they should be using ^

Reply:
unpinned